client/src/components/Navbar/index.tsx

```diff
@@ -1,83 +1,43 @@
-import { useApolloClient } from "@apollo/react-hooks";
 import {
   Alignment,
   Button,
-  Navbar as BPNavbar,
-  Spinner
-} from "@blueprintjs/core";
-import * as QueryString from "query-string";
-import * as React from "react";
-import { Query } from "react-apollo";
-import { useDispatch, useSelector } from "react-redux";
-import useReactRouter from "use-react-router";
+  Navbar as BPNavbar
+} from '@blueprintjs/core';
+import * as React from 'react';
+import {
+  useDispatch,
+  useSelector
+} from 'react-redux';
+import useReactRouter from 'use-react-router';
 
-import Drawer from "./components/Drawer";
-import Header from "./components/Header";
+import Drawer from './drawer';
+import Header from './header';
 
-import { changeNode } from "../../services/selectedNode/actions";
-import { login, logout } from "../../services/user/actions";
+import { logout } from '../../services/user/actions';
```

Comment:
Can we put absolute path?
`import { logout } from 'src/services/user/actions';`

Reply:
done.